src/util/regexp/not-s-and-z.js

```diff
@@ -0,0 +1,24 @@
+define(function() {
+
+/**
+ * Unicode regular expression for: everything except symbols from categories S and Z
+ *
+ * Generated by:
+ *
+ * var s = regenerate()
+ *   .addRange( 0x0, 0x10FFFF )
+ *   .remove( require( "@unicode/unicode-8.0.0/General_Category/Math_Symbol/symbols" ) )
+ *   .remove( require( "@unicode/unicode-8.0.0/General_Category/Currency_Symbol/symbols" ) )
+ *   .remove( require( "@unicode/unicode-8.0.0/General_Category/Modifier_Symbol/symbols" ) )
+ *   .remove( require( "@unicode/unicode-8.0.0/General_Category/Other_Symbol/symbols" ) )
+ *   .remove( require( "@unicode/unicode-8.0.0/General_Category/Space_Separator/symbols" ) )
+ *   .remove( require( "@unicode/unicode-8.0.0/General_Category/Line_Separator/symbols" ) )
+ *   .remove( require( "@unicode/unicode-8.0.0/General_Category/Paragraph_Separator/symbols" ) );
+ *
+ * https://github.com/mathiasbynens/regenerate
+ * https://github.com/node-unicode/unicode-8.0.0
```

Comment:
Here I just followed a common path and I've chosen the v8, but the latest available version of that repo is 13 (https://github.com/node-unicode/unicode-13.0.0). Should I regenerate the regex using v13?

Reply:
Using v13 is ok :+1: